src/DataManagement/Dm.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using ePiggyWeb.DataManagement.Entries;
+using ePiggyWeb.DataManagement.Goals;
+using ePiggyWeb.Utilities;
+
+namespace ePiggyWeb.DataManagement
+{
+    public class Dm
+    {
+        public IEntryManager Income { get; }
+        public IEntryManager Expenses { get; }
+        public IGoalManager Goals { get; }
```

Comment:
What is this class?

Reply:
This somehow got here by accident, was supposed to be stashed away :D It's data manager rework I had started working on